integration-cli/docker_utils.go

```diff
@@ -225,16 +225,19 @@ func (d *Daemon) Start(arg ...string) error {
 	// If we don't explicitly set the log-level or debug flag(-D) then
 	// turn on debug mode
 	foundIt := false
```

Comment:
@coolljt0725 can you do me favor and rename `foundIt` to something like `foundLog`? Since we have more than one "found" variable now I think we need to make "It" be more explicit.